src/queueloader.cpp

```diff
@@ -78,6 +81,22 @@ void queueloader::reload(std::vector<download>& downloads, bool remove_unplayed)
 					utils::tokenize_quoted(line);
 				bool url_found = false;
 
+				if (fields.size() == 0) {
+					if (!comments_ignored) {
+						std::cout << strprintf::fmt(
+							     _("WARNING: Comment found "
+							       "in %s, the queue file is regenerated "
+							       "when podboat exists and comments will "
+							       "be deleted. Press enter to continue or "
+							       "Ctrl+C to abort"),
```

Comment:
1. A typo: "exists" should be "exits".
2. I'd replace first comma with a period, to better separate the problem ("comment found") from its details (an explanation about regeneration).